cob_monitoring/src/cpu_monitor.py

```diff
@@ -189,11 +190,11 @@ def check_uptime(self, interval=1):
             netdata_cpu_load, error = query_netdata('system.load', interval)
```

Comment:
why is the netdata system.uptime stuff mixed with netdata system.load in the first place anyways?
this should be kept separate!

Reply:
Agreed. Thought https://github.com/ipa320/cob_command_tools/blame/indigo_dev/cob_monitoring/src/cpu_monitor.py#L167 makes me wonder if there is a reason for these to be mixed. 
@pgehring, is there a reason for these to be mixed?

`system.load` was added to `check_uptime` in https://github.com/ipa320/cob_command_tools/commit/f01147699f3ccc65731c5a248977fc3f700dd89a#diff-78681ece294d0a1b6ac4749fc4af1a7ebd5ec5ddbd2d695e5799f1cf6d87b4ebR333

Reply:
This is mixed because it was gathered by the `uptime` command in bash. Since it is gathered by netdata it should be seperate - you're right.
There, I sticked too much to the old structure when restructuring it.

Reply:
OK, cool. I already split it up locally, now to test on `mrl-1`